j/jackson-annotations/jackson-annotations_ubi_9.3.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package : jackson-annotations
+# Version : 2.17.0
+# Source repo : https://github.com/FasterXML/jackson-annotations/tree/2.17
+# Tested on : UBI 9.3
+# Language : Java
+# Travis-Check : True
+# Script License: Apache License, Version 2 or later
+# Maintainer : Pratibh Goshi<pratibh.goshi@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ========== platform using the mentioned version of the package.
+# It may not work as expected with newer versions of the
+# package and/or distribution. In such case, please
+# contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+set -e
+
+PACKAGE_NAME="jackson-annotations"
+PACKAGE_VERSION="jackson-annotations-2.17.0"
```

Comment:
please make it like 
`PACKAGE_VERSION=${1:-jackson-annotations-2.17.0}`.. This will allow to use same script for different version through ci pipeline